OpenEdXMobile/src/main/java/org/edx/mobile/view/dialog/CourseModalDialogFragment.kt

```diff
@@ -172,7 +172,6 @@ class CourseModalDialogFragment : DialogFragment() {
         })
 
         iapViewModel.executeOrderResponse.observe(viewLifecycleOwner, NonNullObserver {
-            showUpgradeCompleteDialog()
```

Comment:
better to replace it with the snack bar message for now.

Reply:
updated